In [20]:
# All Libraries required

!pip install numpy==2.0.2
!pip install pandas==2.2.2
!pip install tensorflow_cpu==2.18.0
!pip install scikit-learn==1.3.1

  Using cached numpy-2.0.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (60 kB)
Using cached numpy-2.0.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (19.5 MB)
  Attempting uninstall: numpy
    Found existing installation: numpy 1.26.4
    Uninstalling numpy-1.26.4:
      Successfully uninstalled numpy-1.26.4
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
cupy-cuda12x 12.2.0 requires numpy<1.27,>=1.20, but you have numpy 2.0.2 which is incompatible.
gensim 4.3.3 requires numpy<2.0,>=1.18.5, but you have numpy 2.0.2 which is incompatible.
langchain 0.3.12 requires numpy<2,>=1.22.4; python_version < "3.12", but you have numpy 2.0.2 which is incompatible.
matplotlib 3.8.0 requires numpy<2,>=1.21, but you have numpy 2.0.2 which is incompatible.
pytensor 2.26.4 requires numpy<2,>=1.17.0, but you have numpy 2.0.2 which is incompa

In [21]:
import pandas as pd
import numpy as np
from sklearn.model_selection import train_test_split
from sklearn.metrics import mean_squared_error
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import Dense

In [22]:
# Load dataset
data = pd.read_csv("/content/drive/MyDrive/ML/concrete_data.csv")

In [23]:
# Split features and target
X = data.drop(columns=["Strength"])
y = data["Strength"]

In [24]:
# Normalize the predictors manually using the formula
X_normalized = (X - X.mean()) / X.std()

In [25]:
print(X_normalized)

        Cement  Blast Furnace Slag   Fly Ash     Water  Superplasticizer  \
0     2.476712           -0.856472 -0.846733 -0.916319         -0.620147   
1     2.476712           -0.856472 -0.846733 -0.916319         -0.620147   
2     0.491187            0.795140 -0.846733  2.174405         -1.038638   
3     0.491187            0.795140 -0.846733  2.174405         -1.038638   
4    -0.790075            0.678079 -0.846733  0.488555         -1.038638   
...        ...                 ...       ...       ...               ...   
1025 -0.045623            0.487998  0.564271 -0.092126          0.451190   
1026  0.392628           -0.856472  0.959602  0.675872          0.702285   
1027 -1.269472            0.759210  0.850222  0.521336         -0.017520   
1028 -1.168042            1.307430 -0.846733 -0.279443          0.852942   
1029 -0.193939            0.308349  0.376762  0.891286          0.400971   

      Coarse Aggregate  Fine Aggregate       Age  
0             0.862735       -1.2170

In [27]:
# Initialize list to store MSE values
mse_list = []

# Repeat 50 times
for _ in range(50):
    # 1. Split the data into training and test sets
    X_train, X_test, y_train, y_test = train_test_split(X_normalized, y, test_size=0.3, random_state=np.random.randint(0, 1000))

    # 2. Build the model
    model = Sequential()
    model.add(Dense(10, activation='relu', input_shape=(X_train.shape[1],)))
    model.add(Dense(1))

    # Compile the model
    model.compile(optimizer='adam', loss='mean_squared_error')

    # Train the model
    model.fit(X_train, y_train, validation_data=(X_test, y_test), epochs=50, batch_size=10, verbose=2)

    # 3. Evaluate the model
    y_pred = model.predict(X_test)
    mse = mean_squared_error(y_test, y_pred)

    # Store MSE
    mse_list.append(mse)

Epoch 1/50


/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 15ms/step - loss: 1611.0701 - val_loss: 1568.5889
Epoch 2/50
73/73 - 0s - 2ms/step - loss: 1567.8280 - val_loss: 1528.7734
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1527.4882 - val_loss: 1490.6049
Epoch 4/50
73/73 - 0s - 3ms/step - loss: 1487.9109 - val_loss: 1452.3005
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1446.4122 - val_loss: 1411.6071
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1401.5128 - val_loss: 1368.2194
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 1352.1757 - val_loss: 1320.8867
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 1298.1168 - val_loss: 1269.5966
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 1243.8965 - val_loss: 1218.8467
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 1186.8716 - val_loss: 1164.4242
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 1125.9589 - val_loss: 1106.8511
Epoch 12/50
73/73 - 0s - 5ms/step - loss: 1062.5173 - val_loss: 1046.7490
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 997.7027 - val_loss: 986.2993
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 933.4717 - val_lo

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1545.1467 - val_loss: 1537.7916
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1503.8895 - val_loss: 1499.0433
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1462.6868 - val_loss: 1458.2605
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1418.2286 - val_loss: 1412.5389
Epoch 5/50
73/73 - 0s - 2ms/step - loss: 1368.1012 - val_loss: 1361.0250
Epoch 6/50
73/73 - 0s - 5ms/step - loss: 1311.5533 - val_loss: 1302.7539
Epoch 7/50
73/73 - 0s - 2ms/step - loss: 1247.6073 - val_loss: 1237.4631
Epoch 8/50
73/73 - 0s - 5ms/step - loss: 1176.7529 - val_loss: 1165.5977
Epoch 9/50
73/73 - 0s - 5ms/step - loss: 1099.9087 - val_loss: 1087.6699
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 1019.3335 - val_loss: 1007.4632
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 937.0095 - val_loss: 926.7354
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 854.0012 - val_loss: 845.3598
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 770.8476 - val_loss: 763.0745
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 691.2317 - val_loss: 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1478.2891 - val_loss: 1472.7498
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1424.3809 - val_loss: 1415.9325
Epoch 3/50
73/73 - 0s - 3ms/step - loss: 1367.6901 - val_loss: 1354.9017
Epoch 4/50
73/73 - 0s - 5ms/step - loss: 1306.1781 - val_loss: 1288.2531
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1239.0474 - val_loss: 1215.8230
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1165.3739 - val_loss: 1137.6228
Epoch 7/50
73/73 - 0s - 5ms/step - loss: 1085.5457 - val_loss: 1052.8240
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 1001.4032 - val_loss: 964.7728
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 913.1641 - val_loss: 875.5272
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 824.1636 - val_loss: 785.4709
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 735.8727 - val_loss: 699.1232
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 651.4498 - val_loss: 617.3041
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 572.7501 - val_loss: 542.8680
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 501.3909 - val_loss: 476.4

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 17ms/step - loss: 1515.1860 - val_loss: 1498.6321
Epoch 2/50
73/73 - 0s - 4ms/step - loss: 1465.1683 - val_loss: 1445.6002
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1409.0094 - val_loss: 1384.6270
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1342.9950 - val_loss: 1312.8799
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1264.8757 - val_loss: 1231.6666
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1178.9818 - val_loss: 1143.5811
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 1084.6775 - val_loss: 1049.7300
Epoch 8/50
73/73 - 0s - 5ms/step - loss: 986.9929 - val_loss: 953.1504
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 889.3323 - val_loss: 859.6688
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 792.8546 - val_loss: 767.6233
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 699.6793 - val_loss: 681.3641
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 615.0486 - val_loss: 603.0966
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 538.0237 - val_loss: 531.3994
Epoch 14/50
73/73 - 0s - 3ms/step - loss: 467.1236 - val_loss: 466.42

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 21ms/step - loss: 1594.1841 - val_loss: 1468.2795
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1547.5483 - val_loss: 1422.9253
Epoch 3/50
73/73 - 0s - 5ms/step - loss: 1501.4233 - val_loss: 1376.9000
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1452.7384 - val_loss: 1326.3795
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1398.4380 - val_loss: 1270.9366
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1336.6517 - val_loss: 1208.7845
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 1267.4167 - val_loss: 1139.5421
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 1192.2664 - val_loss: 1066.0725
Epoch 9/50
73/73 - 0s - 3ms/step - loss: 1112.9451 - val_loss: 989.6935
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 1031.5636 - val_loss: 911.6151
Epoch 11/50
73/73 - 0s - 3ms/step - loss: 949.1853 - val_loss: 834.7504
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 866.7221 - val_loss: 756.5960
Epoch 13/50
73/73 - 0s - 5ms/step - loss: 786.0106 - val_loss: 684.3687
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 709.3139 - val_loss: 61

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 20ms/step - loss: 1576.1243 - val_loss: 1574.8448
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1534.0297 - val_loss: 1533.2277
Epoch 3/50
73/73 - 0s - 3ms/step - loss: 1490.3424 - val_loss: 1488.2332
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1442.1982 - val_loss: 1437.4617
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1388.0688 - val_loss: 1379.0992
Epoch 6/50
73/73 - 0s - 3ms/step - loss: 1326.0859 - val_loss: 1312.9564
Epoch 7/50
73/73 - 0s - 5ms/step - loss: 1254.8815 - val_loss: 1235.8168
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 1173.8569 - val_loss: 1150.0538
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 1087.7305 - val_loss: 1060.2625
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 996.8591 - val_loss: 967.5359
Epoch 11/50
73/73 - 0s - 5ms/step - loss: 903.2963 - val_loss: 872.3701
Epoch 12/50
73/73 - 0s - 3ms/step - loss: 810.1728 - val_loss: 780.0475
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 719.7432 - val_loss: 690.7130
Epoch 14/50
73/73 - 0s - 3ms/step - loss: 632.5977 - val_loss: 60

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1503.0137 - val_loss: 1606.5491
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1456.1882 - val_loss: 1559.5093
Epoch 3/50
73/73 - 0s - 3ms/step - loss: 1406.9336 - val_loss: 1506.8099
Epoch 4/50
73/73 - 0s - 2ms/step - loss: 1351.9109 - val_loss: 1447.8772
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1288.1514 - val_loss: 1379.9656
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1217.0621 - val_loss: 1304.2992
Epoch 7/50
73/73 - 0s - 2ms/step - loss: 1140.4257 - val_loss: 1223.4202
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 1059.6315 - val_loss: 1136.4053
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 975.1957 - val_loss: 1048.6895
Epoch 10/50
73/73 - 0s - 5ms/step - loss: 889.2578 - val_loss: 962.2393
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 805.6794 - val_loss: 872.9381
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 724.3925 - val_loss: 789.3802
Epoch 13/50
73/73 - 0s - 5ms/step - loss: 647.7709 - val_loss: 709.2251
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 575.6738 - val_loss: 636

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 15ms/step - loss: 1483.1584 - val_loss: 1493.8600
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1439.6089 - val_loss: 1444.5045
Epoch 3/50
73/73 - 0s - 5ms/step - loss: 1387.7449 - val_loss: 1384.0903
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1322.6001 - val_loss: 1310.1921
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1244.9601 - val_loss: 1224.7644
Epoch 6/50
73/73 - 0s - 3ms/step - loss: 1156.2681 - val_loss: 1128.5021
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 1060.2111 - val_loss: 1026.1049
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 955.9136 - val_loss: 915.0137
Epoch 9/50
73/73 - 0s - 3ms/step - loss: 847.5143 - val_loss: 805.2642
Epoch 10/50
73/73 - 0s - 3ms/step - loss: 740.1909 - val_loss: 696.0654
Epoch 11/50
73/73 - 0s - 5ms/step - loss: 637.4760 - val_loss: 595.0884
Epoch 12/50
73/73 - 0s - 3ms/step - loss: 545.0522 - val_loss: 508.0568
Epoch 13/50
73/73 - 0s - 2ms/step - loss: 464.6475 - val_loss: 431.6481
Epoch 14/50
73/73 - 0s - 3ms/step - loss: 397.7082 - val_loss: 371.23

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 13ms/step - loss: 1583.2351 - val_loss: 1570.2601
Epoch 2/50
73/73 - 0s - 7ms/step - loss: 1539.4830 - val_loss: 1527.2490
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1497.1663 - val_loss: 1484.5946
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1452.7155 - val_loss: 1438.9015
Epoch 5/50
73/73 - 0s - 3ms/step - loss: 1403.5701 - val_loss: 1388.4437
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1348.0674 - val_loss: 1331.0120
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 1287.8030 - val_loss: 1270.5040
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 1222.6246 - val_loss: 1204.8320
Epoch 9/50
73/73 - 0s - 5ms/step - loss: 1153.0599 - val_loss: 1136.0725
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 1080.3518 - val_loss: 1065.0046
Epoch 11/50
73/73 - 0s - 3ms/step - loss: 1005.5235 - val_loss: 992.3513
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 929.6096 - val_loss: 918.7703
Epoch 13/50
73/73 - 0s - 2ms/step - loss: 854.4736 - val_loss: 846.2808
Epoch 14/50
73/73 - 0s - 5ms/step - loss: 779.8115 - val_loss:

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1498.0784 - val_loss: 1554.0829
Epoch 2/50
73/73 - 0s - 7ms/step - loss: 1452.1460 - val_loss: 1506.2389
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1401.5808 - val_loss: 1450.3965
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1343.8551 - val_loss: 1385.7367
Epoch 5/50
73/73 - 0s - 5ms/step - loss: 1277.6055 - val_loss: 1312.6241
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1202.7791 - val_loss: 1230.3750
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 1121.9326 - val_loss: 1141.5729
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 1034.2510 - val_loss: 1045.3004
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 941.2760 - val_loss: 945.3461
Epoch 10/50
73/73 - 0s - 2ms/step - loss: 846.9784 - val_loss: 844.4205
Epoch 11/50
73/73 - 0s - 3ms/step - loss: 752.4748 - val_loss: 745.7150
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 662.8422 - val_loss: 652.2130
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 579.2552 - val_loss: 567.8644
Epoch 14/50
73/73 - 0s - 5ms/step - loss: 502.9528 - val_loss: 494.

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1519.4200 - val_loss: 1571.5708
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1480.6538 - val_loss: 1526.6354
Epoch 3/50
73/73 - 0s - 3ms/step - loss: 1435.0347 - val_loss: 1472.3771
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1378.7607 - val_loss: 1405.0391
Epoch 5/50
73/73 - 0s - 2ms/step - loss: 1310.7501 - val_loss: 1325.5504
Epoch 6/50
73/73 - 0s - 5ms/step - loss: 1231.3868 - val_loss: 1235.2626
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 1142.3119 - val_loss: 1136.6591
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 1047.1266 - val_loss: 1031.6937
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 947.4383 - val_loss: 925.1761
Epoch 10/50
73/73 - 0s - 2ms/step - loss: 846.9845 - val_loss: 817.6096
Epoch 11/50
73/73 - 0s - 5ms/step - loss: 746.4239 - val_loss: 714.7148
Epoch 12/50
73/73 - 0s - 3ms/step - loss: 649.8100 - val_loss: 615.1811
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 562.6447 - val_loss: 529.8417
Epoch 14/50
73/73 - 0s - 5ms/step - loss: 483.2536 - val_loss: 452.

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 13ms/step - loss: 1557.8020 - val_loss: 1546.0652
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1512.1682 - val_loss: 1502.1807
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1464.6224 - val_loss: 1454.1731
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1410.1377 - val_loss: 1399.1052
Epoch 5/50
73/73 - 0s - 2ms/step - loss: 1348.1210 - val_loss: 1337.6866
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1278.9888 - val_loss: 1269.3040
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 1203.1799 - val_loss: 1195.0779
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 1120.4185 - val_loss: 1114.4332
Epoch 9/50
73/73 - 0s - 3ms/step - loss: 1035.8838 - val_loss: 1034.4478
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 948.9246 - val_loss: 949.9147
Epoch 11/50
73/73 - 0s - 5ms/step - loss: 860.7972 - val_loss: 866.3635
Epoch 12/50
73/73 - 1s - 7ms/step - loss: 774.5509 - val_loss: 784.1301
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 690.1563 - val_loss: 704.2202
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 612.0113 - val_loss: 63

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1556.6089 - val_loss: 1470.9025
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1513.5168 - val_loss: 1427.1995
Epoch 3/50
73/73 - 0s - 3ms/step - loss: 1464.7961 - val_loss: 1376.6925
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1407.5077 - val_loss: 1315.6415
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1338.2726 - val_loss: 1242.2964
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1256.9801 - val_loss: 1158.3308
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 1164.3781 - val_loss: 1064.4417
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 1063.2391 - val_loss: 965.6113
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 961.2232 - val_loss: 866.2108
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 863.6215 - val_loss: 772.5305
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 769.5606 - val_loss: 683.5639
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 680.2158 - val_loss: 598.0278
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 599.0451 - val_loss: 522.7263
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 525.3943 - val_loss: 455.3

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 13ms/step - loss: 1498.7833 - val_loss: 1535.1400
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1450.5260 - val_loss: 1483.3844
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1397.7317 - val_loss: 1426.8414
Epoch 4/50
73/73 - 0s - 5ms/step - loss: 1338.4755 - val_loss: 1363.6544
Epoch 5/50
73/73 - 0s - 3ms/step - loss: 1273.1344 - val_loss: 1294.5698
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1202.3931 - val_loss: 1217.1407
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 1125.5824 - val_loss: 1133.6814
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 1041.2435 - val_loss: 1041.7223
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 953.8118 - val_loss: 946.1658
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 864.3174 - val_loss: 849.1795
Epoch 11/50
73/73 - 0s - 3ms/step - loss: 777.5672 - val_loss: 755.7287
Epoch 12/50
73/73 - 0s - 5ms/step - loss: 694.1387 - val_loss: 670.0423
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 616.8873 - val_loss: 586.2191
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 545.8950 - val_loss: 511.

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1613.3223 - val_loss: 1510.0940
Epoch 2/50
73/73 - 0s - 4ms/step - loss: 1573.8264 - val_loss: 1474.6418
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1536.1094 - val_loss: 1440.0133
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1497.6141 - val_loss: 1403.2048
Epoch 5/50
73/73 - 0s - 3ms/step - loss: 1455.2656 - val_loss: 1362.0952
Epoch 6/50
73/73 - 0s - 5ms/step - loss: 1406.8370 - val_loss: 1313.7983
Epoch 7/50
73/73 - 1s - 7ms/step - loss: 1348.8669 - val_loss: 1254.9575
Epoch 8/50
73/73 - 0s - 5ms/step - loss: 1280.7920 - val_loss: 1185.5068
Epoch 9/50
73/73 - 1s - 8ms/step - loss: 1200.8765 - val_loss: 1104.6194
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 1111.9954 - val_loss: 1015.3929
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 1016.5421 - val_loss: 923.0883
Epoch 12/50
73/73 - 0s - 5ms/step - loss: 918.5553 - val_loss: 828.7047
Epoch 13/50
73/73 - 0s - 6ms/step - loss: 820.5652 - val_loss: 735.2043
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 725.2703 - val_loss:

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 20ms/step - loss: 1535.8749 - val_loss: 1604.6429
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1492.6200 - val_loss: 1560.7996
Epoch 3/50
73/73 - 0s - 5ms/step - loss: 1449.2937 - val_loss: 1512.9905
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1401.4635 - val_loss: 1459.9331
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1347.0322 - val_loss: 1398.1285
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1284.5940 - val_loss: 1327.6531
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 1213.3844 - val_loss: 1249.6685
Epoch 8/50
73/73 - 0s - 2ms/step - loss: 1136.8386 - val_loss: 1164.5288
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 1054.6102 - val_loss: 1075.0533
Epoch 10/50
73/73 - 0s - 5ms/step - loss: 967.9487 - val_loss: 979.4549
Epoch 11/50
73/73 - 0s - 3ms/step - loss: 877.5320 - val_loss: 882.3790
Epoch 12/50
73/73 - 0s - 3ms/step - loss: 787.1846 - val_loss: 786.5762
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 699.1481 - val_loss: 695.1589
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 616.2287 - val_loss: 60

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 20ms/step - loss: 1542.5038 - val_loss: 1440.4196
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1493.0300 - val_loss: 1391.6631
Epoch 3/50
73/73 - 0s - 3ms/step - loss: 1440.8701 - val_loss: 1338.3691
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1383.0046 - val_loss: 1279.1375
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1319.2292 - val_loss: 1213.8146
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1248.3246 - val_loss: 1142.5039
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 1172.3768 - val_loss: 1066.7515
Epoch 8/50
73/73 - 0s - 2ms/step - loss: 1088.9237 - val_loss: 983.3383
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 997.9726 - val_loss: 890.8243
Epoch 10/50
73/73 - 0s - 5ms/step - loss: 896.6289 - val_loss: 794.2819
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 791.8414 - val_loss: 696.2057
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 686.7943 - val_loss: 602.1332
Epoch 13/50
73/73 - 0s - 2ms/step - loss: 588.4305 - val_loss: 514.7067
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 497.9211 - val_loss: 437.7

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 18ms/step - loss: 1548.9731 - val_loss: 1642.5438
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1509.6449 - val_loss: 1599.3160
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1472.4290 - val_loss: 1556.7405
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1434.4912 - val_loss: 1512.1477
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1393.0571 - val_loss: 1462.2876
Epoch 6/50
73/73 - 0s - 2ms/step - loss: 1346.0989 - val_loss: 1403.0474
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 1291.3844 - val_loss: 1335.8369
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 1228.1445 - val_loss: 1259.6870
Epoch 9/50
73/73 - 0s - 5ms/step - loss: 1155.4304 - val_loss: 1174.4191
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 1074.1641 - val_loss: 1079.2507
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 984.9140 - val_loss: 980.5609
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 890.9466 - val_loss: 879.3678
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 795.7761 - val_loss: 778.1720
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 702.8813 - val_loss: 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1544.4749 - val_loss: 1524.4359
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1497.0841 - val_loss: 1476.4928
Epoch 3/50
73/73 - 0s - 2ms/step - loss: 1444.1333 - val_loss: 1423.1787
Epoch 4/50
73/73 - 0s - 3ms/step - loss: 1386.1823 - val_loss: 1363.3142
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1320.2269 - val_loss: 1294.8823
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1245.3925 - val_loss: 1218.6646
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 1160.6913 - val_loss: 1133.0526
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 1068.7526 - val_loss: 1040.7787
Epoch 9/50
73/73 - 0s - 2ms/step - loss: 970.0469 - val_loss: 945.2145
Epoch 10/50
73/73 - 0s - 5ms/step - loss: 871.3604 - val_loss: 851.4081
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 776.8134 - val_loss: 762.3436
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 688.2823 - val_loss: 679.9189
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 607.2583 - val_loss: 606.0037
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 535.9786 - val_loss: 541.

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1565.8099 - val_loss: 1615.7238
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1532.1658 - val_loss: 1582.0579
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1497.8608 - val_loss: 1545.6399
Epoch 4/50
73/73 - 0s - 2ms/step - loss: 1459.5940 - val_loss: 1503.3539
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1413.5923 - val_loss: 1449.9822
Epoch 6/50
73/73 - 0s - 3ms/step - loss: 1356.5233 - val_loss: 1385.6017
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 1287.2594 - val_loss: 1309.0863
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 1209.5758 - val_loss: 1222.9543
Epoch 9/50
73/73 - 0s - 3ms/step - loss: 1121.7786 - val_loss: 1126.0974
Epoch 10/50
73/73 - 0s - 2ms/step - loss: 1024.7627 - val_loss: 1023.0493
Epoch 11/50
73/73 - 0s - 2ms/step - loss: 921.8813 - val_loss: 912.2569
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 816.9950 - val_loss: 804.5339
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 715.0240 - val_loss: 699.9249
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 620.3834 - val_loss: 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 15ms/step - loss: 1538.4216 - val_loss: 1458.6984
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1490.2413 - val_loss: 1413.0474
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1438.8641 - val_loss: 1362.5830
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1380.9674 - val_loss: 1305.4120
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1313.4836 - val_loss: 1238.6515
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1234.6744 - val_loss: 1163.2305
Epoch 7/50
73/73 - 0s - 2ms/step - loss: 1145.6387 - val_loss: 1078.7246
Epoch 8/50
73/73 - 0s - 5ms/step - loss: 1049.8813 - val_loss: 990.3613
Epoch 9/50
73/73 - 0s - 3ms/step - loss: 953.1213 - val_loss: 902.3753
Epoch 10/50
73/73 - 0s - 3ms/step - loss: 857.2906 - val_loss: 816.1132
Epoch 11/50
73/73 - 0s - 2ms/step - loss: 763.1708 - val_loss: 731.7443
Epoch 12/50
73/73 - 0s - 3ms/step - loss: 674.9226 - val_loss: 653.5028
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 593.8811 - val_loss: 581.7167
Epoch 14/50
73/73 - 0s - 5ms/step - loss: 520.3409 - val_loss: 517.3

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 15ms/step - loss: 1576.5591 - val_loss: 1528.4783
Epoch 2/50
73/73 - 0s - 2ms/step - loss: 1526.2196 - val_loss: 1476.6810
Epoch 3/50
73/73 - 0s - 5ms/step - loss: 1475.0135 - val_loss: 1423.1860
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1420.6545 - val_loss: 1365.2283
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1361.0472 - val_loss: 1301.7843
Epoch 6/50
73/73 - 0s - 3ms/step - loss: 1296.7712 - val_loss: 1234.1454
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 1227.2155 - val_loss: 1160.9460
Epoch 8/50
73/73 - 0s - 5ms/step - loss: 1152.8816 - val_loss: 1083.0699
Epoch 9/50
73/73 - 0s - 2ms/step - loss: 1075.1243 - val_loss: 1003.5323
Epoch 10/50
73/73 - 0s - 5ms/step - loss: 994.4865 - val_loss: 922.8017
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 914.4343 - val_loss: 841.0888
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 834.7614 - val_loss: 763.0157
Epoch 13/50
73/73 - 0s - 2ms/step - loss: 757.9561 - val_loss: 687.5562
Epoch 14/50
73/73 - 0s - 3ms/step - loss: 683.7078 - val_loss: 61

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1603.3524 - val_loss: 1438.5927
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1553.8152 - val_loss: 1388.3507
Epoch 3/50
73/73 - 0s - 2ms/step - loss: 1503.0011 - val_loss: 1336.3384
Epoch 4/50
73/73 - 0s - 3ms/step - loss: 1449.0970 - val_loss: 1280.6848
Epoch 5/50
73/73 - 0s - 3ms/step - loss: 1391.4984 - val_loss: 1220.6899
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1328.8040 - val_loss: 1155.8806
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 1259.1727 - val_loss: 1085.7833
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 1181.7012 - val_loss: 1011.0705
Epoch 9/50
73/73 - 0s - 5ms/step - loss: 1100.0714 - val_loss: 931.8978
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 1014.5322 - val_loss: 851.6254
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 928.5656 - val_loss: 774.1340
Epoch 12/50
73/73 - 0s - 2ms/step - loss: 843.9870 - val_loss: 698.2982
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 760.4078 - val_loss: 624.7576
Epoch 14/50
73/73 - 0s - 3ms/step - loss: 681.4859 - val_loss: 55

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1474.1270 - val_loss: 1503.4800
Epoch 2/50
73/73 - 0s - 7ms/step - loss: 1425.2336 - val_loss: 1449.7455
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1370.5850 - val_loss: 1387.8204
Epoch 4/50
73/73 - 0s - 3ms/step - loss: 1306.2365 - val_loss: 1315.3571
Epoch 5/50
73/73 - 0s - 5ms/step - loss: 1230.8572 - val_loss: 1229.7059
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1145.7832 - val_loss: 1133.7220
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 1051.4563 - val_loss: 1031.4519
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 951.6213 - val_loss: 925.9951
Epoch 9/50
73/73 - 0s - 5ms/step - loss: 853.5500 - val_loss: 822.5914
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 756.8116 - val_loss: 722.2673
Epoch 11/50
73/73 - 0s - 3ms/step - loss: 664.3797 - val_loss: 627.7529
Epoch 12/50
73/73 - 0s - 7ms/step - loss: 578.0615 - val_loss: 543.1030
Epoch 13/50
73/73 - 1s - 7ms/step - loss: 498.9865 - val_loss: 466.3186
Epoch 14/50
73/73 - 1s - 9ms/step - loss: 430.7858 - val_loss: 401.98

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 15ms/step - loss: 1509.1272 - val_loss: 1561.1404
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1459.6456 - val_loss: 1507.3475
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1408.1884 - val_loss: 1449.6229
Epoch 4/50
73/73 - 0s - 5ms/step - loss: 1352.4374 - val_loss: 1385.8488
Epoch 5/50
73/73 - 1s - 8ms/step - loss: 1290.5372 - val_loss: 1317.8844
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1222.8506 - val_loss: 1241.5848
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 1150.2424 - val_loss: 1161.1166
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 1072.5345 - val_loss: 1074.8293
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 989.6689 - val_loss: 985.8719
Epoch 10/50
73/73 - 0s - 3ms/step - loss: 904.4020 - val_loss: 895.3413
Epoch 11/50
73/73 - 0s - 5ms/step - loss: 818.9427 - val_loss: 805.4640
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 733.5539 - val_loss: 717.6808
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 652.1894 - val_loss: 634.6898
Epoch 14/50
73/73 - 1s - 9ms/step - loss: 577.3416 - val_loss: 559.

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 22ms/step - loss: 1551.0511 - val_loss: 1610.1263
Epoch 2/50
73/73 - 0s - 5ms/step - loss: 1512.8677 - val_loss: 1571.0480
Epoch 3/50
73/73 - 1s - 8ms/step - loss: 1475.7717 - val_loss: 1529.0144
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1434.3240 - val_loss: 1480.4988
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1387.2164 - val_loss: 1424.2937
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1333.1296 - val_loss: 1360.3936
Epoch 7/50
73/73 - 0s - 5ms/step - loss: 1271.2516 - val_loss: 1289.3588
Epoch 8/50
73/73 - 1s - 10ms/step - loss: 1202.8201 - val_loss: 1211.7869
Epoch 9/50
73/73 - 0s - 5ms/step - loss: 1127.6857 - val_loss: 1126.9663
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 1049.0461 - val_loss: 1041.6757
Epoch 11/50
73/73 - 0s - 3ms/step - loss: 968.5866 - val_loss: 954.2542
Epoch 12/50
73/73 - 0s - 3ms/step - loss: 886.2405 - val_loss: 866.8292
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 805.0291 - val_loss: 783.6700
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 728.9223 - val_loss:

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1547.6455 - val_loss: 1472.6666
Epoch 2/50
73/73 - 1s - 7ms/step - loss: 1504.2771 - val_loss: 1431.6229
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1460.0853 - val_loss: 1387.9673
Epoch 4/50
73/73 - 0s - 5ms/step - loss: 1412.2635 - val_loss: 1339.8523
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1356.8358 - val_loss: 1282.2522
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1290.6294 - val_loss: 1214.3568
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 1212.9705 - val_loss: 1134.6064
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 1123.6661 - val_loss: 1045.8679
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 1026.2981 - val_loss: 951.2032
Epoch 10/50
73/73 - 0s - 5ms/step - loss: 922.9880 - val_loss: 854.9570
Epoch 11/50
73/73 - 0s - 3ms/step - loss: 820.5359 - val_loss: 759.2978
Epoch 12/50
73/73 - 0s - 2ms/step - loss: 720.3251 - val_loss: 668.9958
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 625.6953 - val_loss: 584.7063
Epoch 14/50
73/73 - 0s - 5ms/step - loss: 542.2929 - val_loss: 511

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1623.8458 - val_loss: 1461.5693
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1574.5916 - val_loss: 1414.9254
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1523.6853 - val_loss: 1365.3971
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1468.5544 - val_loss: 1310.8157
Epoch 5/50
73/73 - 0s - 5ms/step - loss: 1407.7827 - val_loss: 1250.8888
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1340.7026 - val_loss: 1186.6868
Epoch 7/50
73/73 - 0s - 2ms/step - loss: 1268.1694 - val_loss: 1117.5168
Epoch 8/50
73/73 - 0s - 5ms/step - loss: 1191.1823 - val_loss: 1045.4329
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 1110.3857 - val_loss: 972.5869
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 1027.5288 - val_loss: 898.2223
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 943.4542 - val_loss: 822.3911
Epoch 12/50
73/73 - 0s - 2ms/step - loss: 861.3377 - val_loss: 751.9774
Epoch 13/50
73/73 - 0s - 5ms/step - loss: 782.7651 - val_loss: 683.1257
Epoch 14/50
73/73 - 0s - 3ms/step - loss: 707.9153 - val_loss: 61

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1642.1930 - val_loss: 1470.1205
Epoch 2/50
73/73 - 0s - 2ms/step - loss: 1593.5879 - val_loss: 1428.3999
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1550.2644 - val_loss: 1390.4169
Epoch 4/50
73/73 - 0s - 3ms/step - loss: 1509.5789 - val_loss: 1354.2526
Epoch 5/50
73/73 - 0s - 3ms/step - loss: 1470.6082 - val_loss: 1319.0448
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1430.6559 - val_loss: 1281.8662
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 1387.6940 - val_loss: 1242.5411
Epoch 8/50
73/73 - 0s - 2ms/step - loss: 1343.8553 - val_loss: 1201.2356
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 1295.5259 - val_loss: 1154.8077
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 1240.5020 - val_loss: 1099.5468
Epoch 11/50
73/73 - 0s - 2ms/step - loss: 1174.6599 - val_loss: 1033.2627
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 1098.5972 - val_loss: 958.9104
Epoch 13/50
73/73 - 0s - 5ms/step - loss: 1013.4624 - val_loss: 878.9323
Epoch 14/50
73/73 - 0s - 3ms/step - loss: 925.4362 - val_lo

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 13ms/step - loss: 1479.0426 - val_loss: 1533.0781
Epoch 2/50
73/73 - 0s - 2ms/step - loss: 1431.3145 - val_loss: 1481.0663
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1377.9182 - val_loss: 1422.7473
Epoch 4/50
73/73 - 0s - 5ms/step - loss: 1315.6694 - val_loss: 1353.2742
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1244.4769 - val_loss: 1274.8237
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1163.8619 - val_loss: 1186.9612
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 1075.2800 - val_loss: 1092.4829
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 981.1678 - val_loss: 989.7452
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 885.5646 - val_loss: 890.5817
Epoch 10/50
73/73 - 0s - 2ms/step - loss: 792.0406 - val_loss: 792.4595
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 701.8480 - val_loss: 697.2997
Epoch 12/50
73/73 - 0s - 3ms/step - loss: 615.6555 - val_loss: 607.1766
Epoch 13/50
73/73 - 0s - 2ms/step - loss: 536.8453 - val_loss: 526.4882
Epoch 14/50
73/73 - 0s - 5ms/step - loss: 466.3620 - val_loss: 453.43

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1480.0477 - val_loss: 1571.0304
Epoch 2/50
73/73 - 0s - 2ms/step - loss: 1435.2098 - val_loss: 1524.0320
Epoch 3/50
73/73 - 0s - 3ms/step - loss: 1382.7894 - val_loss: 1467.1504
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1319.9469 - val_loss: 1400.5758
Epoch 5/50
73/73 - 0s - 2ms/step - loss: 1249.5168 - val_loss: 1325.6526
Epoch 6/50
73/73 - 0s - 5ms/step - loss: 1170.5122 - val_loss: 1242.2825
Epoch 7/50
73/73 - 0s - 2ms/step - loss: 1083.5275 - val_loss: 1150.7296
Epoch 8/50
73/73 - 0s - 2ms/step - loss: 992.0762 - val_loss: 1055.9410
Epoch 9/50
73/73 - 0s - 2ms/step - loss: 895.7060 - val_loss: 958.6084
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 802.3824 - val_loss: 863.0020
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 711.2453 - val_loss: 769.9636
Epoch 12/50
73/73 - 0s - 3ms/step - loss: 623.9538 - val_loss: 680.9037
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 542.5330 - val_loss: 598.7448
Epoch 14/50
73/73 - 0s - 5ms/step - loss: 468.7281 - val_loss: 524.2

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


Epoch 1/50
73/73 - 1s - 16ms/step - loss: 1591.0907 - val_loss: 1464.5726
Epoch 2/50
73/73 - 0s - 2ms/step - loss: 1546.2902 - val_loss: 1419.9113
Epoch 3/50
73/73 - 0s - 5ms/step - loss: 1495.7095 - val_loss: 1367.7759
Epoch 4/50
73/73 - 0s - 6ms/step - loss: 1435.8513 - val_loss: 1305.9100
Epoch 5/50
73/73 - 1s - 8ms/step - loss: 1365.3751 - val_loss: 1234.3062
Epoch 6/50
73/73 - 0s - 5ms/step - loss: 1285.3219 - val_loss: 1153.0299
Epoch 7/50
73/73 - 1s - 8ms/step - loss: 1194.6124 - val_loss: 1064.0548
Epoch 8/50
73/73 - 0s - 5ms/step - loss: 1096.7513 - val_loss: 969.2635
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 994.2238 - val_loss: 871.7911
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 892.8981 - val_loss: 777.2711
Epoch 11/50
73/73 - 0s - 5ms/step - loss: 794.0408 - val_loss: 688.3229
Epoch 12/50
73/73 - 1s - 7ms/step - loss: 700.9129 - val_loss: 603.8349
Epoch 13/50
73/73 - 1s - 7ms/step - loss: 614.4171 - val_loss: 529.6201
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 537.5582 - val_

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 2s - 23ms/step - loss: 1551.4230 - val_loss: 1561.6429
Epoch 2/50
73/73 - 0s - 5ms/step - loss: 1503.7690 - val_loss: 1519.1981
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1458.1554 - val_loss: 1476.9709
Epoch 4/50
73/73 - 1s - 9ms/step - loss: 1410.6484 - val_loss: 1430.5446
Epoch 5/50
73/73 - 1s - 7ms/step - loss: 1358.4502 - val_loss: 1379.5050
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1300.3762 - val_loss: 1322.4027
Epoch 7/50
73/73 - 0s - 5ms/step - loss: 1235.9532 - val_loss: 1259.2786
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 1165.2606 - val_loss: 1190.1660
Epoch 9/50
73/73 - 0s - 3ms/step - loss: 1090.5635 - val_loss: 1119.1345
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 1011.3919 - val_loss: 1040.4237
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 928.7180 - val_loss: 962.1389
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 847.7665 - val_loss: 882.4847
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 767.0425 - val_loss: 804.7406
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 689.0472 - val_loss: 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 15ms/step - loss: 1569.3358 - val_loss: 1485.4608
Epoch 2/50
73/73 - 0s - 2ms/step - loss: 1521.6658 - val_loss: 1441.2986
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1473.1375 - val_loss: 1394.5035
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1420.2367 - val_loss: 1341.8049
Epoch 5/50
73/73 - 0s - 3ms/step - loss: 1360.4767 - val_loss: 1281.5634
Epoch 6/50
73/73 - 0s - 3ms/step - loss: 1293.3921 - val_loss: 1215.4602
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 1218.4855 - val_loss: 1141.0619
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 1135.6602 - val_loss: 1059.7969
Epoch 9/50
73/73 - 0s - 2ms/step - loss: 1046.3069 - val_loss: 973.3878
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 952.4165 - val_loss: 884.3076
Epoch 11/50
73/73 - 0s - 5ms/step - loss: 857.7801 - val_loss: 796.7296
Epoch 12/50
73/73 - 0s - 2ms/step - loss: 766.0243 - val_loss: 711.8550
Epoch 13/50
73/73 - 0s - 5ms/step - loss: 678.2482 - val_loss: 631.3278
Epoch 14/50
73/73 - 0s - 3ms/step - loss: 594.4548 - val_loss: 556

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 15ms/step - loss: 1530.6251 - val_loss: 1594.8524
Epoch 2/50
73/73 - 0s - 2ms/step - loss: 1489.8279 - val_loss: 1552.1677
Epoch 3/50
73/73 - 0s - 3ms/step - loss: 1446.5338 - val_loss: 1505.9819
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1398.4813 - val_loss: 1453.0209
Epoch 5/50
73/73 - 0s - 5ms/step - loss: 1342.8474 - val_loss: 1390.2238
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1276.5345 - val_loss: 1316.8937
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 1202.1542 - val_loss: 1234.1942
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 1117.6566 - val_loss: 1140.5177
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 1025.7529 - val_loss: 1043.6736
Epoch 10/50
73/73 - 0s - 3ms/step - loss: 929.2620 - val_loss: 939.8887
Epoch 11/50
73/73 - 0s - 2ms/step - loss: 830.1861 - val_loss: 838.5903
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 734.0441 - val_loss: 737.0071
Epoch 13/50
73/73 - 0s - 5ms/step - loss: 643.4211 - val_loss: 644.7242
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 560.8469 - val_loss: 56

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1552.8563 - val_loss: 1446.0497
Epoch 2/50
73/73 - 0s - 2ms/step - loss: 1501.2358 - val_loss: 1397.1121
Epoch 3/50
73/73 - 0s - 3ms/step - loss: 1447.8213 - val_loss: 1344.6643
Epoch 4/50
73/73 - 0s - 2ms/step - loss: 1388.3613 - val_loss: 1285.5298
Epoch 5/50
73/73 - 0s - 5ms/step - loss: 1321.9475 - val_loss: 1221.2896
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1250.5294 - val_loss: 1152.4779
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 1172.7000 - val_loss: 1076.7843
Epoch 8/50
73/73 - 0s - 2ms/step - loss: 1088.9104 - val_loss: 997.1929
Epoch 9/50
73/73 - 0s - 2ms/step - loss: 1001.8646 - val_loss: 913.8611
Epoch 10/50
73/73 - 0s - 3ms/step - loss: 912.0522 - val_loss: 828.6360
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 822.0521 - val_loss: 744.5721
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 733.7758 - val_loss: 661.2601
Epoch 13/50
73/73 - 0s - 5ms/step - loss: 648.0551 - val_loss: 580.8823
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 568.5482 - val_loss: 506.

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 15ms/step - loss: 1554.3140 - val_loss: 1607.7257
Epoch 2/50
73/73 - 0s - 7ms/step - loss: 1514.3348 - val_loss: 1564.4739
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1471.6724 - val_loss: 1515.6799
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1423.2965 - val_loss: 1459.8054
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1366.9608 - val_loss: 1394.9318
Epoch 6/50
73/73 - 0s - 5ms/step - loss: 1299.9938 - val_loss: 1320.4849
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 1224.4510 - val_loss: 1234.9352
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 1139.4906 - val_loss: 1143.2615
Epoch 9/50
73/73 - 0s - 3ms/step - loss: 1048.2971 - val_loss: 1046.6219
Epoch 10/50
73/73 - 0s - 5ms/step - loss: 952.8288 - val_loss: 946.4575
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 855.9565 - val_loss: 844.4508
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 759.5204 - val_loss: 746.0461
Epoch 13/50
73/73 - 0s - 5ms/step - loss: 668.0540 - val_loss: 652.1042
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 581.6456 - val_loss: 56

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1534.2004 - val_loss: 1486.1615
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1482.0586 - val_loss: 1435.9349
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1426.2645 - val_loss: 1379.9985
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1363.4622 - val_loss: 1317.6730
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1294.2528 - val_loss: 1247.9889
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1217.0258 - val_loss: 1173.9829
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 1134.7561 - val_loss: 1093.3859
Epoch 8/50
73/73 - 0s - 5ms/step - loss: 1049.2435 - val_loss: 1010.7428
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 961.8046 - val_loss: 928.4659
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 874.8167 - val_loss: 844.2602
Epoch 11/50
73/73 - 0s - 5ms/step - loss: 786.8946 - val_loss: 762.1646
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 705.0460 - val_loss: 685.4443
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 630.2338 - val_loss: 613.4854
Epoch 14/50
73/73 - 0s - 5ms/step - loss: 561.6793 - val_loss: 549.

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1519.8899 - val_loss: 1528.7069
Epoch 2/50
73/73 - 0s - 7ms/step - loss: 1478.7585 - val_loss: 1484.9500
Epoch 3/50
73/73 - 0s - 5ms/step - loss: 1435.0306 - val_loss: 1436.5704
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1385.9100 - val_loss: 1380.4836
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1328.6993 - val_loss: 1314.9159
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1261.5836 - val_loss: 1240.1019
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 1185.8430 - val_loss: 1157.2308
Epoch 8/50
73/73 - 0s - 5ms/step - loss: 1104.9783 - val_loss: 1071.3173
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 1020.0734 - val_loss: 982.0734
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 933.8563 - val_loss: 891.6806
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 847.0834 - val_loss: 804.2209
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 761.6752 - val_loss: 716.7693
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 678.7557 - val_loss: 633.8516
Epoch 14/50
73/73 - 0s - 3ms/step - loss: 599.9091 - val_loss: 554

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 19ms/step - loss: 1667.9562 - val_loss: 1486.7478
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1626.5450 - val_loss: 1450.7650
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1583.0585 - val_loss: 1409.3647
Epoch 4/50
73/73 - 0s - 5ms/step - loss: 1532.5082 - val_loss: 1360.5031
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1471.4514 - val_loss: 1301.9983
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1399.6396 - val_loss: 1233.9229
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 1317.0601 - val_loss: 1157.7356
Epoch 8/50
73/73 - 0s - 5ms/step - loss: 1226.9287 - val_loss: 1077.0577
Epoch 9/50
73/73 - 1s - 8ms/step - loss: 1132.9945 - val_loss: 991.8423
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 1033.9725 - val_loss: 904.5060
Epoch 11/50
73/73 - 0s - 2ms/step - loss: 932.8188 - val_loss: 816.0523
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 831.7479 - val_loss: 728.4224
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 733.3337 - val_loss: 646.0831
Epoch 14/50
73/73 - 0s - 3ms/step - loss: 644.1110 - val_loss: 57

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 19ms/step - loss: 1576.2373 - val_loss: 1631.5146
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1536.8210 - val_loss: 1592.1395
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1499.7194 - val_loss: 1550.6204
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1460.8193 - val_loss: 1506.2076
Epoch 5/50
73/73 - 0s - 3ms/step - loss: 1418.1062 - val_loss: 1455.8800
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1369.9176 - val_loss: 1398.7059
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 1315.0332 - val_loss: 1336.6448
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 1254.8507 - val_loss: 1269.0364
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 1189.4436 - val_loss: 1197.5795
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 1119.5143 - val_loss: 1123.6317
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 1045.6350 - val_loss: 1045.7029
Epoch 12/50
73/73 - 0s - 5ms/step - loss: 970.1613 - val_loss: 967.6829
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 895.7692 - val_loss: 891.1495
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 822.1460 - val_loss

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 15ms/step - loss: 1496.6519 - val_loss: 1500.2599
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1446.3046 - val_loss: 1446.9189
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1387.5575 - val_loss: 1383.4537
Epoch 4/50
73/73 - 0s - 3ms/step - loss: 1318.1068 - val_loss: 1312.3134
Epoch 5/50
73/73 - 0s - 2ms/step - loss: 1240.4426 - val_loss: 1234.0575
Epoch 6/50
73/73 - 0s - 5ms/step - loss: 1156.1808 - val_loss: 1148.2327
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 1065.5397 - val_loss: 1056.7982
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 971.2723 - val_loss: 962.7089
Epoch 9/50
73/73 - 0s - 2ms/step - loss: 875.2715 - val_loss: 867.7917
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 780.0602 - val_loss: 773.5102
Epoch 11/50
73/73 - 0s - 5ms/step - loss: 687.7990 - val_loss: 682.4631
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 601.7746 - val_loss: 595.6866
Epoch 13/50
73/73 - 0s - 3ms/step - loss: 521.1486 - val_loss: 515.5930
Epoch 14/50
73/73 - 0s - 3ms/step - loss: 449.2490 - val_loss: 445.03

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1528.5740 - val_loss: 1529.2953
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1483.0686 - val_loss: 1484.5546
Epoch 3/50
73/73 - 0s - 3ms/step - loss: 1434.9792 - val_loss: 1435.1848
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1380.6710 - val_loss: 1379.3108
Epoch 5/50
73/73 - 0s - 3ms/step - loss: 1319.1829 - val_loss: 1315.2325
Epoch 6/50
73/73 - 0s - 3ms/step - loss: 1249.0457 - val_loss: 1242.6515
Epoch 7/50
73/73 - 0s - 3ms/step - loss: 1171.4148 - val_loss: 1160.6733
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 1083.1819 - val_loss: 1069.8380
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 985.1566 - val_loss: 972.1401
Epoch 10/50
73/73 - 0s - 2ms/step - loss: 884.9832 - val_loss: 871.1526
Epoch 11/50
73/73 - 0s - 2ms/step - loss: 784.3090 - val_loss: 772.7674
Epoch 12/50
73/73 - 0s - 3ms/step - loss: 688.9500 - val_loss: 678.8859
Epoch 13/50
73/73 - 0s - 3ms/step - loss: 599.8006 - val_loss: 592.2966
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 518.6968 - val_loss: 512.

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 15ms/step - loss: 1483.4542 - val_loss: 1500.3728
Epoch 2/50
73/73 - 0s - 7ms/step - loss: 1437.2815 - val_loss: 1448.3423
Epoch 3/50
73/73 - 0s - 3ms/step - loss: 1378.8646 - val_loss: 1385.4138
Epoch 4/50
73/73 - 0s - 3ms/step - loss: 1309.8929 - val_loss: 1311.5500
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1229.4696 - val_loss: 1226.6235
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1139.1029 - val_loss: 1132.0641
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 1040.9625 - val_loss: 1032.9758
Epoch 8/50
73/73 - 0s - 5ms/step - loss: 939.1296 - val_loss: 930.9719
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 836.9583 - val_loss: 829.7369
Epoch 10/50
73/73 - 0s - 3ms/step - loss: 736.9152 - val_loss: 733.5458
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 641.8174 - val_loss: 643.8091
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 557.2759 - val_loss: 565.2535
Epoch 13/50
73/73 - 0s - 2ms/step - loss: 482.5592 - val_loss: 493.9232
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 418.0388 - val_loss: 435.13

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1564.1345 - val_loss: 1603.5353
Epoch 2/50
73/73 - 0s - 2ms/step - loss: 1519.4341 - val_loss: 1559.0792
Epoch 3/50
73/73 - 0s - 5ms/step - loss: 1476.2058 - val_loss: 1514.4882
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1431.8011 - val_loss: 1467.0068
Epoch 5/50
73/73 - 0s - 3ms/step - loss: 1384.1906 - val_loss: 1415.3306
Epoch 6/50
73/73 - 0s - 3ms/step - loss: 1332.1536 - val_loss: 1359.0576
Epoch 7/50
73/73 - 0s - 4ms/step - loss: 1273.9885 - val_loss: 1295.1088
Epoch 8/50
73/73 - 0s - 5ms/step - loss: 1208.9946 - val_loss: 1223.9454
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 1137.0094 - val_loss: 1145.8260
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 1058.4750 - val_loss: 1061.7302
Epoch 11/50
73/73 - 0s - 2ms/step - loss: 977.3069 - val_loss: 976.2226
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 894.7963 - val_loss: 888.0720
Epoch 13/50
73/73 - 0s - 3ms/step - loss: 812.6654 - val_loss: 801.6641
Epoch 14/50
73/73 - 0s - 2ms/step - loss: 731.9496 - val_loss: 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 16ms/step - loss: 1554.3169 - val_loss: 1560.8848
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1510.4509 - val_loss: 1518.4827
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1468.1775 - val_loss: 1474.9803
Epoch 4/50
73/73 - 0s - 3ms/step - loss: 1423.3511 - val_loss: 1428.0793
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1373.1957 - val_loss: 1372.2565
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1315.8644 - val_loss: 1307.4861
Epoch 7/50
73/73 - 0s - 5ms/step - loss: 1250.0513 - val_loss: 1234.0073
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 1176.4133 - val_loss: 1152.4259
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 1096.8915 - val_loss: 1066.5239
Epoch 10/50
73/73 - 0s - 3ms/step - loss: 1012.7628 - val_loss: 977.9350
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 926.1496 - val_loss: 888.5791
Epoch 12/50
73/73 - 0s - 5ms/step - loss: 838.8480 - val_loss: 799.6542
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 753.7404 - val_loss: 714.1870
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 671.4601 - val_loss: 6

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1499.7611 - val_loss: 1506.3049
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1451.0740 - val_loss: 1451.2567
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1395.7183 - val_loss: 1387.1305
Epoch 4/50
73/73 - 0s - 4ms/step - loss: 1330.7063 - val_loss: 1313.8771
Epoch 5/50
73/73 - 0s - 5ms/step - loss: 1255.5107 - val_loss: 1230.7750
Epoch 6/50
73/73 - 0s - 3ms/step - loss: 1168.8772 - val_loss: 1136.3723
Epoch 7/50
73/73 - 0s - 2ms/step - loss: 1072.5103 - val_loss: 1033.0162
Epoch 8/50
73/73 - 0s - 3ms/step - loss: 970.3057 - val_loss: 926.8112
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 865.4973 - val_loss: 822.0526
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 763.4080 - val_loss: 721.3580
Epoch 11/50
73/73 - 0s - 5ms/step - loss: 666.6311 - val_loss: 625.8276
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 576.9485 - val_loss: 541.4479
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 497.4230 - val_loss: 466.0247
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 426.9018 - val_loss: 401.32

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 14ms/step - loss: 1531.8877 - val_loss: 1601.7559
Epoch 2/50
73/73 - 0s - 6ms/step - loss: 1483.9347 - val_loss: 1549.2633
Epoch 3/50
73/73 - 0s - 2ms/step - loss: 1436.5225 - val_loss: 1495.3163
Epoch 4/50
73/73 - 0s - 2ms/step - loss: 1388.7465 - val_loss: 1440.2054
Epoch 5/50
73/73 - 0s - 2ms/step - loss: 1338.3737 - val_loss: 1382.0848
Epoch 6/50
73/73 - 0s - 3ms/step - loss: 1285.1780 - val_loss: 1319.4401
Epoch 7/50
73/73 - 0s - 2ms/step - loss: 1227.9999 - val_loss: 1252.8059
Epoch 8/50
73/73 - 0s - 2ms/step - loss: 1166.1013 - val_loss: 1180.4095
Epoch 9/50
73/73 - 0s - 3ms/step - loss: 1098.6403 - val_loss: 1103.2444
Epoch 10/50
73/73 - 0s - 6ms/step - loss: 1025.5940 - val_loss: 1019.6358
Epoch 11/50
73/73 - 1s - 9ms/step - loss: 946.4711 - val_loss: 930.1313
Epoch 12/50
73/73 - 1s - 8ms/step - loss: 858.6827 - val_loss: 830.0850
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 766.7531 - val_loss: 732.1254
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 675.4412 - val_loss: 

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 13ms/step - loss: 1502.3527 - val_loss: 1655.6436
Epoch 2/50
73/73 - 0s - 3ms/step - loss: 1463.5128 - val_loss: 1615.3153
Epoch 3/50
73/73 - 0s - 4ms/step - loss: 1424.3208 - val_loss: 1572.2953
Epoch 4/50
73/73 - 0s - 2ms/step - loss: 1380.3381 - val_loss: 1523.6796
Epoch 5/50
73/73 - 0s - 4ms/step - loss: 1330.3491 - val_loss: 1465.3354
Epoch 6/50
73/73 - 0s - 4ms/step - loss: 1272.0239 - val_loss: 1400.3958
Epoch 7/50
73/73 - 0s - 5ms/step - loss: 1208.9021 - val_loss: 1329.0452
Epoch 8/50
73/73 - 0s - 4ms/step - loss: 1139.6652 - val_loss: 1251.0477
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 1066.5259 - val_loss: 1169.5723
Epoch 10/50
73/73 - 0s - 5ms/step - loss: 991.4630 - val_loss: 1085.6117
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 912.9504 - val_loss: 998.6187
Epoch 12/50
73/73 - 0s - 3ms/step - loss: 833.0930 - val_loss: 905.2042
Epoch 13/50
73/73 - 0s - 4ms/step - loss: 751.5689 - val_loss: 813.8513
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 668.9111 - val_loss: 7

/usr/local/lib/python3.10/dist-packages/keras/src/layers/core/dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


73/73 - 1s - 13ms/step - loss: 1607.0442 - val_loss: 1550.8175
Epoch 2/50
73/73 - 0s - 2ms/step - loss: 1565.6843 - val_loss: 1511.7050
Epoch 3/50
73/73 - 0s - 2ms/step - loss: 1527.5953 - val_loss: 1473.3007
Epoch 4/50
73/73 - 0s - 5ms/step - loss: 1487.3602 - val_loss: 1430.5292
Epoch 5/50
73/73 - 0s - 5ms/step - loss: 1442.0433 - val_loss: 1382.8512
Epoch 6/50
73/73 - 1s - 9ms/step - loss: 1390.8674 - val_loss: 1329.4481
Epoch 7/50
73/73 - 1s - 7ms/step - loss: 1333.8829 - val_loss: 1270.1796
Epoch 8/50
73/73 - 1s - 8ms/step - loss: 1270.7788 - val_loss: 1206.4579
Epoch 9/50
73/73 - 0s - 4ms/step - loss: 1203.7524 - val_loss: 1138.8389
Epoch 10/50
73/73 - 0s - 4ms/step - loss: 1132.0387 - val_loss: 1068.5089
Epoch 11/50
73/73 - 0s - 4ms/step - loss: 1059.4281 - val_loss: 998.0486
Epoch 12/50
73/73 - 0s - 4ms/step - loss: 985.7983 - val_loss: 926.4683
Epoch 13/50
73/73 - 0s - 5ms/step - loss: 911.8772 - val_loss: 855.8533
Epoch 14/50
73/73 - 0s - 4ms/step - loss: 840.4398 - val_loss:

In [28]:
# 4. Calculate mean and standard deviation of MSE
mean_mse = np.mean(mse_list)
std_mse = np.std(mse_list)

In [29]:
# 5. Print results after normalization
print(f"Mean of MSE over 50 runs: {mean_mse}")
print(f"Standard Deviation of MSE over 50 runs: {std_mse}")

Mean of MSE over 50 runs: 130.31662939683622
Standard Deviation of MSE over 50 runs: 11.836501926494584


In [7]:
#  Print results in previous one before normalization
print(f"Mean of MSE over 50 runs: {mean_mse}")
print(f"Standard Deviation of MSE over 50 runs: {std_mse}")

Mean of MSE over 50 runs: 117.68993547247621
Standard Deviation of MSE over 50 runs: 24.892526714211833


#Mean of MSE:
###Before normalization: 117.69
###After normalization: 130.32
The mean MSE increased after normalization. This indicates that normalization did not improve the model's performance in this case, as a lower MSE is preferred.

#Standard Deviation of MSE:
###Before normalization: 24.89
###After normalization: 11.84
The standard deviation decreased after normalization, indicating that the results are more consistent (less variability) across the 50 runs.